packages/gatsby-theme-newrelic/src/components/ContributingGuidelines.js

```diff
@@ -95,6 +104,8 @@ const ContributingGuidelines = ({ fileRelativePath }) => {
 
 ContributingGuidelines.propTypes = {
   fileRelativePath: PropTypes.string,
+  title: PropTypes.string,
+  slug: PropTypes.string,
```

Comment:
The more I think about it, we might be able to get this data internally and avoid this prop.

Gatsby uses [`@reach/router`](https://reach.tech/router/) under the hood. This exports a `useLocation` hook that you can use to get the current page location. Perhaps we could take advantage of this?

```js
import { useLocation } from '@reach/router'

// This should be the same as `slug`
const { pathname } = useLocation()
```

The less props to worry about, the better! This feedback would also apply to the `GlobalFooter`

Reply:
Oh that's fantastic. I'll definitely incorporate that.